x/group/internal/orm/auto_uint64.go

```diff
@@ -73,3 +73,36 @@ func (a AutoUInt64Table) GetOne(store sdk.KVStore, rowID uint64, dest codec.Prot
 	}
 	return rawRowID, nil
 }
+
+// PrefixScan returns an Iterator over a domain of keys in ascending order. End is exclusive.
+// Start is an MultiKeyIndex key or prefix. It must be less than end, or the Iterator is invalid and error is returned.
+// Iterator must be closed by caller.
+// To iterate over entire domain, use PrefixScan(nil, nil)
```

Comment:
Is this possible? The start and end arguments are not pointers so you can't use nil AFAICS

Reply:
good catch, in the case of AutoUInt64Table, we need to use proper integers, in this case, that would be `PrefixScan(1, math.MaxUint64), docs updated